addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13859,7 +13869,7 @@ msgstr ""
 
 #: xbmc/peripherals/devices/peripheralcecadapter.cpp
 msgctxt "#36017"
-msgid "Could not initialise the CEC adaptor: libCEC was not found on your system."
+msgid "Couldn't initialise the CEC adaptor: libCEC was not found on your system."
```

Comment:
So I'll go with primary spelling choice here...  Just because it reminds me of some ah-hem other words ending with "bator"